### DISCUSSION TITLE
How to use fsx suite to test MEMFS?

### DISCUSSION THREAD

**Discussion by Lucasq11** [winfsp/winfsp#521] [Q&A]
As https://winfsp.dev/doc/WinFsp-Testing/ introduces, winfsp has several test suites. I only found winfsp-tests in source code, how to use the other suites like fsx, Fscrash, and Fsbench?

**Comment by billziss-gh** [comment]
You can find those test suites in this repo: https://github.com/billziss-gh/secfs.test

You can see how WinFsp does it in its master test driver: https://github.com/winfsp/winfsp/blob/master/tools/run-tests.bat
